lib/bowling.rb

```diff
@@ -0,0 +1,30 @@
+class BowlingScore
+  attr_reader :current_score
+
+  def initialize
+    @number_of_rolls = 0
```

Comment:
like the use of incremental number of rolls to keep track of how many shots have been taken